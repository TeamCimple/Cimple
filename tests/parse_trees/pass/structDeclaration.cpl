struct Employee {
	int age;
	int salary;
};

int main(){
	int * e;

	e = make Employee(a, b, c);
	return 0;
}
