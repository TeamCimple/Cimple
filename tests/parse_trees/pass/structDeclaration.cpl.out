Struct((int Var(age),), (int Var(salary),), , Employee, , )FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN(Int(0)))) 
